adapters/between/betweentest/exemplary/multi-request.json

```diff
@@ -0,0 +1,127 @@
+{
+  "mockBidRequest": {
+    "id": "some-request-id1",
+    "imp": [
+      {
+        "id": "some-impression-id1",
+        "banner": {
+          "format": [
+            {
+              "h": 250,
+              "w": 300
+            }
+          ],
+          "h": 250,
+          "w": 300
+        },
+        "ext": {
+          "bidder": {
+            "host": "127.0.0.2"
+          }
+        }
+      },
+      {
+        "id": "some-impression-id2",
+        "banner": {
+          "format": [
+            {
+              "h": 250,
+              "w": 300
+            }
+          ],
+          "h": 250,
+          "w": 300
+        },
+        "ext": {
+          "bidder": {
+            "host": "127.0.0.2"
+          }
+        }
+      }
+    ],
+    "device": {
+      "ip": "8.8.8.12"
+    },
+    "tmax": 500
+  },
+  "httpCalls": [
+
+    {
+      "expectedRequest": {
+        "uri": "http://127.0.0.2/",
+        "body": {
+          "id": "some-request-id1",
+
+          "imp": [
+            {
+              "id": "some-impression-id1",
```

Comment:
Shouldn't the expected request to your service have both the impressions in it?

Reply:
done, pushing soon